doc/introduction/introducing_zephyr.rst

```diff
@@ -29,8 +30,42 @@ as described in :ref:`Zephyr_Licensing`.
 Distinguishing Features
 ***********************
 
-The Zephyr kernel offers a number of features that distinguish it from other
-small-footprint OSes:
+Zephyr offers a large and evergrowing number of features including:
+
+**Extensive suite of Kernel services**
+   Zephyr offers a number of familiar services for development:
+
+   * *Multi-threading Services* for cooperative, priority-based,
+     non-preemptive, and preemptive threads with optional round robin
+     time-slicing. Includes POSIX pthreads compatible API support.
+
+   * *Interrupt Services* for compile-time registration of interrupt handlers.
+
+   * *Memory Allocation Services* for dynamic allocation and freeing of
+     fixed-size or variable-size memory blocks.
+
+   * *Inter-thread Synchronization Services* for binary semaphores,
+     counting semaphores, and mutex semaphores.
+
+   * *Inter-thread Data Passing Services* for basic message queues, enhanced
+     message queues, and byte streams.
+
+   * *Power Management Services* such as tickless idle and an advanced idling
+     infrastructure.
+
+**Multiple Scheduling Algorithms**
+
```

Comment:
This section is a series of high-level topics using a reST dictionary list.  To have this topic look like the rest of the topics, it needs to start with an indented sentence to contain the nested bullet points. So, change this to:

    **Multiple Scheduling Algorithms**

       Zephyr provides a comprehensive set of thread scheduling choices:

       * Cooperative and Preemptive Scheduling